Tasks/Stansky/Task7/Task 7_5.py

```diff
@@ -0,0 +1,30 @@
+cache = {}
```

Comment:
but keep in mind that such a cache would be shared between all the decorated functions